src/features/wallet/root/walletLayout.tsx

```diff
@@ -4,6 +4,7 @@ import React from 'react'
 import { SafeAreaInsetsContext } from 'react-native-safe-area-context'
 import { wh, hp } from '../../../utils/layout'
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const withWalletLayout = (WrappedComponent: any) => (props: any) => {
```

Comment:
I think we could clean up one of these any's like this
```ts
export const withWalletLayout = (WrappedComponent: React.ComponentType) => (
  props: any,
) => 
```

Reply:
It breaks where you use it to wrap a component. I also tried a bunch of others such as `Element`, `JSX.Element`, `ReactElement` with no success :/